modules/python36/module.yaml

```diff
@@ -0,0 +1,5 @@
+schema_version: 1
+
```

Comment:
hmm, these module.yaml files get copied into the build directories, but afaik they are used by concreate and not by the docker build. Maybe we should move them somewhere else to keep them out of the build image?  This is not new, the existing openshift-spark does this too.

@crobby am I crazy here?